Cargo.toml

```diff
@@ -8,3 +8,4 @@ edition = "2021"
 [dependencies]
 yultsur = { git = "https://github.com/leonardoalt/yultsur", branch = "main" }
 tempfile = "3"
+clap = "2.26.2"
```

Comment:
What is the review policy for adding external dependencies? Yolo?

Reply:
It's curated, I copied from ZoKrates.

Reply:
Ask @Schaeff what his policy was